### ISSUE TITLE
Package 'ncdf4.helpers' could not be installed

### ISSUE THREAD

**Issue by lloyddewit** [africanmathsinitiative/R-Instat#5768]
**Describe the bug**
When I started a new R-Instat build (equal to the current main branch), I got the following message:

![image](https://user-images.githubusercontent.com/57253949/77528456-49282300-6e8e-11ea-9272-d83a8af96eb8.png)

I clicked 'Install Missing Packages (Online)' and got the message:

![image](https://user-images.githubusercontent.com/57253949/77632026-b353cd00-6f4d-11ea-8f9b-2375a6e2f016.png)

The missing package means that Issue #5766 cannot be recreated.

**To Reproduce**
I think it's only possible to reproduce this behaviour on platforms that don't yet have `ncdf4.helpers` installed. It's possible that some R-Instat developers installed this package before CRAN archived it on 16-11-2019.
Manually uninstalling this package should also allow the issue to be reproduced.

**Expected behaviour**
I expect if the package if it's missing, then R-Instat installs it automatically. Or maybe an alternative to `ncdf4.helpers` could be found?

**Additional context**
CRAN states [here](https://cran.r-project.org/web/packages/ncdf4.helpers/index.html): 

`Package ‘ncdf4.helpers’ was removed from the CRAN repository. Formerly available versions can be obtained from the archive. `

A workaround is to install the package from the CRAN archive. To do this, run the following commands in the R-Instat script window:

```
packageurl <- "http://cran.r-project.org/src/contrib/Archive/ncdf4.helpers/ncdf4.helpers_0.3-3.tar.gz"
install.packages(packageurl, repos=NULL, type="source")
test1 <- require("ncdf4.helpers")
test1
```

The last 2 lines are just to confirm that the package was installed. If `test1` is true then the install was successful.

If I restart R-Instat then I still get an error message saying that the pacakage is not installed, but the commands needed to recreate Issue #5766 now work correctly.

**Comment by dannyparsons** [comment]
This package was removed from CRAN. We still needed it so we created a modified version. Our version is available here https://drive.google.com/file/d/1UqwqVDxMvlHNocUa0i6uiGFT24zwGdgB/view?usp=sharing (and described in the guide for creating the installer https://github.com/africanmathsinitiative/R-Instat/blob/master/installer/Installer_Generation_Guide.md). It can be downloaded and installed in R/RStudio from the file. Then it should be available in R-Instat.
It is included in the installed version of R-Instat so users shouldn't have this problem.
I think I shared this with other developers before but maybe not with you, sorry!
